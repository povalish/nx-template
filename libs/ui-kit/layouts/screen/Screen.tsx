import React from 'react';
import { screen } from './Screen.classes';
import { IWebScreen } from './Screen.interface';

//
//

export const Screen: React.FC<IWebScreen> = ({ className, children }) => (
  <div className={screen({ className })}>{children}</div>
);
