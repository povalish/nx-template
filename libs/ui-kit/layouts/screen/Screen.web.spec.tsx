import React from 'react';
import { render } from '@testing-library/react';
import { Screen } from './Screen';

//
//

describe('UI Kit - Layout - Screen component', () => {
  it('should render successfully', () => {
    const { baseElement } = render(<Screen>Content</Screen>);
    expect(baseElement).toMatchSnapshot();
  });
});
