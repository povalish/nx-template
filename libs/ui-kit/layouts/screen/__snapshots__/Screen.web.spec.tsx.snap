// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Screen layout should render successfully 1`] = `
<body>
  <div>
    <div
      class="
  w-full h-full bg-colorBackground
  transition-colors
"
    >
      Content
    </div>
  </div>
</body>
`;
